codecov.yml

```diff
@@ -1,34 +1,39 @@
 codecov:
-  disable_default_path_fixes: yes
+  disable_default_path_fixes: true
```

Comment:
Also since we've disabled the default path fixes we probably need to specify our paths manually

Reply:
let me try removing this line, this has always been disabled, but worth a try